rewrite-core/src/main/java/org/openrewrite/config/DeclarativeRecipe.java

```diff
@@ -355,7 +356,7 @@ private static class NameEmail {
     }
 
     @Override
-    public List<Contributor> getContributors() {
+    public synchronized List<Contributor> getContributors() {
```

Comment:
but if the problem is single threaded the change to the collection should be in this code block, I do not see that issue here.

Since this issue pops up in a futureTask I think it makes sense to assume that the visitor in the future is accessing this getter at the same time as some other piece might be using the getter.